docs/content/en/docs/resources/_index.md

```diff
@@ -24,11 +24,21 @@ description: "Additional resources to learn about Tetragon"
 
 ### Blog posts
 
+[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```

Comment:
```suggestion
[Can I Use Tetragon without Cilium? Yes!](https://isovalent.com/blog/post/can-i-use-tetragon-without-cilium-yes/) - [Dean Lewis](https://twitter.com/saintdle), 2023
```
For some reason, Twitter is making the link checker timeout and I would not rely on Twitter from now on... Is it okay if we remove this link?
```
"url": "https://twitter.com/saintdle",
        "status": {
          "text": "Timeout"
}
```

Reply:
done

Reply:
I think link is still there https://github.com/cilium/tetragon/compare/63f2051b9d32b8ecf47566b835759d09435a9df1..302b6b6978651e4bdae17d4759227860363e43f2 you just changed your commit message. I think you might have forgot to `git add` sorry I skipped that in the process. You need a blank line before the "Signed-off" thingie as well.